nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/add", (req, res) => {
+  const firstNumber = Number(req.query.firstParam);
+  const secondNumber = Number(req.query.secondParam);
+  if (firstNumber == 0) {
```

Comment:
Why number 0 is not valid?

The only case where this is problem is division by zero. 
You need to handle NaN in your code.